src/components/Main/Main.jsx

```diff
@@ -0,0 +1,28 @@
+import {useState, useEffect, useContext} from 'react'
+import './Main.css'
+import steamSVG from "../../../public/assets/images/steam.svg";
+import {AnimatePresence, motion} from "framer-motion";
+function Main({navigate}) {
+
+  const handleButtonClick = () => navigate('/libraries');
+
+  return (
+    
+      <div className="main">
+            <h2 className='main__subtitle'>NEW GAME?</h2>
+        <h2 className="main__continue" onClick={handleButtonClick}>click to continue</h2>
```

Comment:
* The hierarchy of headings must be correct

You have `h2` in line 12 for the heading, but here it should be a `button` because you click it, according to the semantics